test/runtime/signal.c

```diff
@@ -45,15 +45,20 @@ static void yield_for(volatile int * v)
         fail_error("timed out");
 }
 
-static void * tgkill_test_pause(void * arg)
+static void * tgkill_test_pause(void *use_select)
 {
+    int rv;
     child_tid = syscall(SYS_gettid);
     sigtest_debug("child enter, tid %d\n", child_tid);
+    if (use_select) {
+        rv = syscall(SYS_pselect6, 0, NULL, NULL, NULL, NULL);
```

Comment:
pselect() has 6 parameters (nfds, readfds, writefds, exceptfds, timeout, sigmask), here we have 5 arguments.

Reply:
Thanks, fixed.